packages/table/preview/localStore.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ *
+ * Demonstrates sample usage of the table component.
+ */
+
+import { defaults } from "lodash";
```

Comment:
I see `lodash` is only a dev dependency, but still, do we really need it for this one function? Would prefer not to pull it in if we've gone this long without it.

Alternatively, we could just cherry-pick the function we need, as mentioned in their docs:

```tsx
// Cherry-pick methods for smaller browserify/rollup/webpack bundles.
var at = require('lodash/at');
var curryN = require('lodash/fp/curryN');
```

Reply:
not important for preview, but i'll switch it

Reply:
when I switched to `import * as defaults from "lodash/defaults"` the types couldn't be resolved.